core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20241121_billing_account_changes_status.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.30.xsd">
```

Comment:
of note: I'm using the latest xsd here - `dbchangelog-4.30.xsd` - instead of the ancient `dbchangelog-3.4.xsd` in other changesets. No particular reason other than we should probably stay current.